## Data used in the course

Use individual level data [1000Genomes phase 1](https://www.cog-genomics.org/plink/1.9/resources) and a skimmed version of
phase 3 is [LDREF](https://data.broadinstitute.org/alkesgroup/FUSION/LDREF.tar.bz2). We would also
[simulate via GCTA](http://cnsgenomics.com/software/gcta/#GWASSimulation).

Foulkes et al. provides [real data](https://www.mtholyoke.edu/courses/afoulkes/Data/GWAStutorial/GWASTutorial_Files.zip) used in their 
tutorial for GWAS using R. The data will be used in Day 1 and Day 3.

The BMI GWAS summary statistics (Locke, et al. 2015) is available from the GIANT website and examine the file.
```
wget http://portals.broadinstitute.org/collaboration/giant/images/1/15/SNP_gwas_mc_merge_nogc.tbl.uniq.gz
```
and you can examine its content as follows,
```
gunzip -c SNP_gwas_mc_merge_nogc.tbl.uniq.gz | more
```
[Table 4 in the Supplementary information](https://www.nature.com/nature/journal/v518/n7538/extref/nature14177-s2.xlsx) has he information required.
In particular the data will be used in Day 1, Day2, Day 4 and Day 5.

## Summary statistics

[NHGRI-EBI list](https://www.ebi.ac.uk/gwas/downloads/summary-statistics) of studies with summary statistics is a companion to the [GWAS catalog](http://www.ebi.ac.uk/gwas/).

## Individual level data

[dbGaP](https://www.ncbi.nlm.nih.gov/gap). `The database of Genotypes and Phenotypes (dbGaP) was developed to archive and distribute the data and results from studies that have investigated the interaction of genotype and phenotype in Humans.`

[EGA](https://www.ebi.ac.uk/ega/home). `The European Genome-phenome Archive (EGA) allows you to explore datasets from genomic studies, provided by a range of data providers. Access to datasets must be approved by the specified Data Access Committee (DAC).`

[GAW](https://www.gaworkshop.org/). `The Genetic Analysis Workshops (GAWs) are a collaborative effort among genetic epidemiologists to evaluate and compare statistical genetic methods. For each GAW, topics are chosen that are relevant to current analytical problems in genetic epidemiology, and sets of real or computer-simulated data are distributed to investigators worldwide.  Results of analyses are discussed and compared at meetings held in even-numbered years.`

[ExAC](http://exac.broadinstitute.org/) is the Exome Aggregation Consortium, a coalition of investigators seeking to aggregate and harmonize exome sequencing data from a wide variety of large-scale sequencing projects, and to make summary data available.

[1000 Genomes project](http://www.internationalgenome.org/) contains information on available data, software and publications.

[The Haplotype Reference Consortium](http://www.haplotype-reference-consortium.org/) creates a large reference panel of human haplotypes by combining together sequencing data from multiple cohorts.

## Genes and genetic disorders

[GeneCards](http://www.genecards.org/) s a searchable, integrative database that provides comprehensive, user-friendly information on all annotated and predicted human genes. It automatically integrates gene-centric data 
from ~125 web sources, including genomic, transcriptomic, proteomic, genetic, clinical and functional information.

[KEGG](http://www.genome.jp/kegg/) or Kyoto Encyclopedia of Genes and Genomes is a database resource for understanding high-level functions and utilities of the biological system, such as the cell, the organism and the ecosystem, from molecular-level information, especially large-scale molecular datasets generated by genome sequencing and other high-throughput experimental technologies.

[MSigDB](http://software.broadinstitute.org/gsea/msigdb/collections.jsp) The Molecular Signatures Database are divided into 8 major collections and several sub-collections on 17779 gene sets.

[PANTHER](http://pantherdb.org/) now includes comprehensive GO annotations directly imported from the GO database.

[REACTOME](http://www.reactome.org/) is a free, open-source, curated and peer reviewed pathway database.

[OMIM](http://www.omim.org/) is an online catalog of human genes and genetic disorders. [MetaMap](https://mmtx.nlm.nih.gov/) is a tool for recognizing UMLS concepts in text. MetaMap is one of the foundations of NLM's 
Medical Text Indexer ([MTI](https://ii.nlm.nih.gov/MTI/)) which is being used for both semiautomatic and fully automatic indexing of biomedical literature at NLM

## Integration of expression, methylation and protein data

[Gene ontology](http://www.geneontology.org/) provides controlled vocabularies of defined terms representing gene product properties. These cover three domains: Cellular Component, the parts of a cell or its extracellular environment; Molecular Function, the elemental activities of a gene product at the molecular level, such as binding or catalysis; and Biological Process, operations or sets of molecular events with a defined beginning and end, pertinent to the functioning of integrated living units: cells, tissues, organs, and organisms.

[GTEx](https://gtexportal.org/home/) provides a platform to study human gene expression and regulation and its relationship to genetic variation. The relevant 
software is [GTEx-pipeline](https://github.com/broadinstitute/gtex-pipeline).

[Regulatorycircuits](http://regulatorycircuits.org/index.html) is a repository of human cell type- and tissue-specific gene regulatory networks.

[ROADMAP and ENDCODE epigenomics data](http://egg2.wustl.edu/roadmap/web_portal/) has information for a total of 127 consolidated epigenomes.

[STRING](https://string-db.org/) is dedicated to protein-protein interaction networks.

## Comparative genomics and animal models

[Emsembl](http://www.ensembl.org/index.html) is a genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation. Ensembl annotate genes, computes multiple alignments, predicts regulatory function and collects disease data. Ensembl tools include BLAST, BLAT, BioMart and the Variant Effect Predictor (VEP) for all supported species.

[IMPC](http://www.mousephenotype.org) is the International Mouse Phenotyping Consortium database.

[MGI](http://www.informatics.jax.org) is the Mouse Genome Informatics database.

[NCBI](https://www.ncbi.nlm.nih.gov/) is a U.S. government-funded national resource for molecular biology information.

[UCSC](https://genome.ucsc.edu/) includes a broad collection of vertebrate and model organism assemblies and annotations, along with a large suite of tools for viewing, analyzing and downloading data. 

